pkg/version/version.go

```diff
@@ -113,8 +106,8 @@ func getVersionBundle(version string) (VersionBundle, error) {
 		MulticlusterControlplane: "v0.7.0",
 	}
 
-	versionBundleList["0.16.1"] = VersionBundle{
-		OCM:                      "v0.16.1",
+	versionBundleList["1.0.0"] = VersionBundle{
+		OCM:                      "v1.0.0",
 		AppAddon:                 "v0.16.0",
```

Comment:
I will remove this line in the future. I am pretty sure it's not being used anymore.